### ISSUE TITLE
如何在预训练模型的基础上继续finetune

### ISSUE THREAD

**Issue by coolboyqu** [PaddlePaddle/PaddleOCR#8962]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem
我想在预训练模型的基础上继续训练几个模糊的字的识别，字典也没有改，数据量很小，但是训练完成后只能识别我训练的几个字，其他的字都无法识别了，感觉像是全部从头训练了，请问我该怎么做可以在预训练模型的基础上继续训练啊

这是我的配置文件，只改了这几个地方
Global:
  debug: false
  use_gpu: true
  epoch_num: 30
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output/rec_ppocr_v3_distillation
  save_epoch_step: 3
  eval_batch_step: [0, 2000]
  cal_metric_during_train: true
  pretrained_model: ./pretrain_models/ch_PP-OCRv3_rec_train/best_accuracy
  # pretrained_model: ./output/rec_ppocr_v3_distillation/latest
  checkpoints:
  save_inference_dir: ./inference/ch_PP-OCRv3_rec/
  use_visualdl: false
  infer_img: doc/imgs_words/ch/word_1.jpg
  character_dict_path: ppocr/utils/ppocr_keys_v1.txt
  max_text_length: &max_text_length 25
  infer_mode: false
  use_space_char: true
  distributed: true
  save_res_path: ./output/rec/predicts_ppocrv3_distillation.txt

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./
    ext_op_transform_idx: 1
    label_file_list:
    - ./rec_gt_train.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - RecConAug:
        prob: 0.5
        ext_data_num: 2
        image_shape: [48, 320, 3]
        max_text_length: *max_text_length
    - RecAug:
    - MultiLabelEncode:
    - RecResizeImg:
        image_shape: [3, 48, 320]
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_sar
        - length
        - valid_ratio
  loader:
    shuffle: true
    batch_size_per_card: 8
    drop_last: true
    num_workers: 4
Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./
    label_file_list:
    - ./rec_gt_test.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - MultiLabelEncode:
    - RecResizeImg:
        image_shape: [3, 48, 320]
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_sar
        - length
        - valid_ratio
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 4
    num_workers: 4

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

**Comment by an1018** [comment]
可以参考这个教程进行识别模型finetune：https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_ch/finetune.md
